src/tokens/watchman/light.json

```diff
@@ -99,27 +67,12 @@
     "text-color-informational": {
       "value": "{neutral colors.neutral-00}"
     },
-    "text-color-informational-inverse": {
+    "image-caption-text-color-inverse": {
       "value": "{neutral colors.neutral-100}"
     }
   },
   "icons": {
     "type": "color",
-    "icon-color-primary": {
-      "value": "{neutral colors.neutral-00}"
-    },
-    "icon-color-action": {
-      "value": "{action-primary colors.action-primary-20}"
-    },
-    "icon-color-subtle": {
-      "value": "{neutral colors.neutral-20}"
-    },
-    "icon-color-disabled": {
-      "value": "{neutral colors.neutral-20}"
-    },
```

Comment:
icon-color-disabled might be a problem. we had a plan to remove that token and use invert 40%, but the values are 40/60/40/20